message.go

```diff
@@ -174,11 +174,16 @@ type MessageFlags int
 const (
 	MessageFlagsCrossPosted          MessageFlags = 1 << 0
 	MessageFlagsIsCrossPosted        MessageFlags = 1 << 1
-	MessageFlagsSupressEmbeds        MessageFlags = 1 << 2
+	MessageFlagsSuppressEmbeds       MessageFlags = 1 << 2
 	MessageFlagsSourceMessageDeleted MessageFlags = 1 << 3
 	MessageFlagsUrgent               MessageFlags = 1 << 4
 )
 
+// Deprecated: use MessageFlagsSuppressEmbeds
+const (
+	MessageFlagsSupressEmbeds MessageFlags = 1 << 2
+)
+
```

Comment:
The reason I put it separate was because of a gofmt error in [the previous build](https://app.travis-ci.com/github/bwmarrin/discordgo/jobs/526140295)

```
0.06s$ diff <(gofmt -d .) <(echo -n)
1,14d0
< diff -u message.go.orig message.go
< --- message.go.orig	2021-07-22 08:05:45.890441692 +0000
< +++ message.go	2021-07-22 08:05:45.890441692 +0000
< @@ -172,8 +172,8 @@
<  
<  // Valid MessageFlags values
<  const (
< -	MessageFlagsCrossPosted          MessageFlags = 1 << 0
< -	MessageFlagsIsCrossPosted        MessageFlags = 1 << 1
< +	MessageFlagsCrossPosted   MessageFlags = 1 << 0
< +	MessageFlagsIsCrossPosted MessageFlags = 1 << 1
<  	// Deprecated: use MessageFlagsSuppressEmbeds
<  	MessageFlagsSupressEmbeds        MessageFlags = 1 << 2
<  	MessageFlagsSuppressEmbeds       MessageFlags = 1 << 2
The command "diff <(gofmt -d .) <(echo -n)" exited with 1.
```

Between messing up the spacing or extracting it to a separate section I chose the latter.

Reply:
Just put it at the end of the section, like that
```go
const (
   ...
   // TODO: remove when compatibility is not required
   MessageFlagsSupressEmbeds       MessageFlags = 1 << 2
)
```